### PR TITLE
Fixed issue #6770

### PR DESCRIPTION
**Pull request by UmashankarTriforce**
Fixed issue #6770 by adding a print prompt before stdin

### CODE REVIEW

**Comment by UmashankarTriforce** [comment]
+sudo apt-get install -y openmpi-bin openmpi-common libopenmpi-dev
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package libopenmpi-dev is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Unable to locate package openmpi-bin
E: Unable to locate package openmpi-common
E: Package 'libopenmpi-dev' has no installation candidate
The command "bash scripts/ci/travis/run-tests.sh before_install" failed and exited with 100 during .

Your build has been stopped.






>How do I fix this issue caused by script while running on ubuntu1604?

File: examples/text_classification/run_text_classifier.py [hunk 1]
```diff
@@ -36,6 +36,7 @@ def setup_model(device, model_setup):
 
 def run_online(device):
     # predict labels online
+    print("Enter inputs for Online Predictions")
```
Comment (hvy) [inline examples/text_classification/run_text_classifier.py]: Could you use single quotes, per convention?

File: examples/text_classification/run_text_classifier.py [hunk 2]
```diff
@@ -66,6 +67,7 @@ def predict_batch(words_batch):
             print('{}\t{:.4f}\t{}'.format(answer, score, ' '.join(words)))
 
     batch = []
+    print("Enter inputs for Batch Predictions")
```
Comment (hvy) [inline examples/text_classification/run_text_classifier.py]: Ditto.

**Comment by hvy** [comment]
Thanks for the PR. Could you take a look at my comments above? The failure seem unrelated to your changes. Let me rerun the job and see if it persists.

File: examples/text_classification/run_text_classifier.py (hunk 1 above)
Comment (UmashankarTriforce) [reply examples/text_classification/run_text_classifier.py]: Okay I'll change to single quotes and commit again

**Comment by hvy** [comment]
Jenkins, test this please.

**Comment by hvy** [comment]
Thanks for the PR. It LGTM but let me check the CI.

**Comment by hvy** [comment]
Jenkins, test this please.

**Comment by hvy** [comment]
Jenkins, test this please.

**Comment by hvy** [comment]
Let me change the name of the title of this PR to reflect the concrete changes.

**Pull request merged**